pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -236,13 +246,17 @@ effectiveStdenv.mkDerivation rec {
       ../include/onnxruntime/core/framework/provider_options.h \
       ../include/onnxruntime/core/providers/cpu/cpu_provider_factory.h \
       ../include/onnxruntime/core/session/onnxruntime_*.h
+  '' + lib.optionalString openvinoSupport ''
+    # hack to allow onnxruntime to find openvino libraries
+    ln -s ${lib.getLib openvino}/runtime/lib/*/libopenvino{,_c,_auto*,_intel*,_hetero*,_onnx*}.so.* $out/lib/
```

Comment:
We could maybe use patchelf here instead

Reply:
Fixing the cmake install dirs of openvino resolved this.